tags/cncf-tags.md

```diff
@@ -26,61 +26,96 @@ TOC and TOC Contributors have fulfilled TAG duties in the past and will continue
 
 ### TAG App-Delivery
 #### Chairs
-* [Lian Li](https://github.com/lianmakesthings)
-* [Josh Gavant](https://github.com/joshgav)
-* [Thomas Schuetz](https://github.com/thschue)
+| Name                  | Organization            | Term                |
```

Comment:
We'd like to introduce and implement formal terms for all TAG leads but it's still being discussed as part of the leadership ladder initiative.

Let's remove this column from here and we can reconsider this suggestion as part of this initiative.

We can keep this PR only focused towards TAG Network related changes.

Reply:
Remove the terms, kept the columns.